pages/1.12/cli/configure/index.md

```diff
@@ -32,3 +32,70 @@ To configure a proxy for the CLI:
 *   Define `no_proxy` for domains that you don’t want to use the proxy for:
 
         export no_proxy=".mesos,.thisdcos.directory,.dcos.directory,.zk,127.0.0.1,localhost,foo.bar.com,.baz.com”
+
+# Environment variables
+
+The DC/OS CLI supports the following environment variables, which can be set dynamically.
+
+<a name="dcos-cluster"></a>
+#### `DCOS_CLUSTER` (DC/OS CLI 0.5.x and higher only)
+
+The [attached](/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/) cluster. To set the attached cluster, set the variable with the command:
```

Comment:
Replace 
"The [attached](/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/) cluster. To set the [attached](/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/), set the variable with the command:" 

with
 "To set the  [attached cluster](/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/), you must set the variable with the command: